grpc.go

```diff
@@ -934,6 +945,39 @@ func (a *agentGRPC) SignalProcess(ctx context.Context, req *pb.SignalProcessRequ
 	return emptyResp, nil
 }
 
+// Check is the container process installed the
+// handler for specific signal.
+func isSignalHandled(pid int, signum syscall.Signal) bool {
+	var termMask uint64 = 1 << (uint(signum) - 1)
+	procFile := fmt.Sprintf("/proc/%d/status", pid)
+	file, err := os.Open(procFile)
+	if err != nil {
+		agentLog.Warnf("Open proc file %s failed", procFile)
+		return false
+	}
+	defer file.Close()
+
+	scanner := bufio.NewScanner(file)
+	for scanner.Scan() {
+		line := scanner.Text()
+		if strings.HasPrefix(line, "SigCgt:") {
+			maskSlice := strings.Split(line, ":")
+			if len(maskSlice) != 2 {
+				agentLog.Warnf("Parse the SigCgt field in proc file %s failed", procFile)
+				return false
+			}
+			sigCgtStr := strings.TrimSpace(maskSlice[1])
+			sigCgtMask, err := strconv.ParseUint(sigCgtStr, 16, 64)
+			if err != nil {
+				agentLog.Warnf("parse the sigCgtStr %s to hex failed", sigCgtStr)
```

Comment:
```go
agentLog.WithField("sigCgt", sigCgtStr).Warn("parse the sigCgt to hex failed")
```

Reply:
Updated

Reply:
thanks, the last one https://github.com/kata-containers/agent/pull/526/files#diff-727ccd4217b7f335e689be03bfb69e9bR955

Reply:
Done !